10. Databases/2.solution_n2

```diff
@@ -0,0 +1,6 @@
+INSERT INTO users1 (first_name, second_name, age) VALUES ('Dima', 'Tsiupko', 24);
```

Comment:
nit: it can be done without duplication like that:
```
INSERT INTO users (first_name, last_name, age)
VALUES ('Dima', 'Tsiupko', 24),
       ('Lera', 'Shabalinska', 24),
       ...
```